influxdb/tests/py26_test.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+import sys
+import os
+import datetime
+import requests
+import time
+
+from influxdb import InfluxDBClient
+
+# InfluxDB Server parameters
+influxdb_server = "localhost"
+port = 8086
+database_name = "test_db"
+ip_addr = os.popen("hostname").read()
```

Comment:
This is somewhat problematic, you should use the socket infra instead of `popen`.

```
In [1]: import socket

In [2]: ip = socket.gethostbyname(socket.gethostname())

In [3]: ip
Out[3]: '192.168.1.77'
```

Reply:
I will use socket.gethostname() because socket.gethostbyname(socket.gethostname()) tends to give the loopback interface ip address this value is pulled from the /etc/hosts file by default on stuck configuration 127.0.0.1 is associated to the hostname